StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheet+DeferredAPI.swift

```diff
@@ -17,6 +17,8 @@ extension PaymentSheet {
         authenticationContext: STPAuthenticationContext,
         paymentHandler: STPPaymentHandler,
         isFlowController: Bool,
+        isDashboardApp: Bool,
+        dashboardParamsUpdater: @escaping DashboardParamsUpdater = setParamsForDashboardApp(confirmType:paymentIntentParams:paymentIntent:configuration:),
```

Comment:
Will this ever be anything other than the default value? If we don't have a case where we need to pass in a different updater here, I would just remove it and call the default function directly

Reply:
> Will this ever be anything other than the default value?

Nope! The only reason I did this was for testing so we can test that the `dashboardParamsUpdater` is called under the correct circumstances. https://github.com/stripe/stripe-ios/pull/2984/files#diff-d596d94b7af6c551f33f81a8feceef006d2345d8af7718e26fff50f52e22cbbeR108